tests/Test_1dtube_cxx/Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG from=precice/precice-ubuntu1604.home-develop
+FROM $from
+USER root
+
+RUN apt-get -qq update && apt-get -qq install \
+    liblapack-dev && \
+    rm -rf /var/lib/apt/lists/*
+
+
+ARG CACHEBUST
+
+RUN mkdir -p /Output/Postproc
+
+
+WORKDIR /home/precice/
+RUN git clone https://github.com/Eder-K/elastictube1d.git
+
+WORKDIR /home/precice/elastictube1d/elastictube1d-cxx
+RUN cmake . && make all && ./Allrun-cxx
```

Comment:
The path and filename are outdated. [This](https://github.com/precice/elastictube1d/pull/16/files#diff-086639b9f75f9dc62d57bce5880881ba) is the new file used in the elastictube1d PR.